src/components/SetHorVertDistanceModal.tsx

```diff
@@ -17,25 +18,27 @@ export const GetInfoModal = ({
   isSegNameEditable,
   value: initialValue,
   initialVariableName,
-}: {
-  isOpen: boolean
-  onResolve: (a: {
+}: InstanceProps<
+  {
     value: string
     segName: string
     valueNode: Value
     variableName?: string
     newVariableInsertIndex: number
     sign: number
-  }) => void
-  onReject: (a: any) => void
+  },
+  boolean
+> & {
   segName: string
   isSegNameEditable: boolean
-  value: number
+  value?: number
   initialVariableName: string
 }) => {
   const [sign, setSign] = useState(Math.sign(Number(initialValue)))
   const [segName, setSegName] = useState(initialSegName)
-  const [value, setValue] = useState(String(Math.abs(initialValue)))
+  const [value, setValue] = useState(
+    initialValue ? String(Math.abs(initialValue)) : ''
+  )
```

Comment:
This is where the potential bug was found. `initialValue` here was expected to be provided as a `number`, but once the types were resolved, I noticed the value being provided was nullable, because it comes from the return of `transformSecondarySketchLinesTagFirst`.

So I defaulted the value here to empty string if `initialValue` is falsey. Please let me know if that's alright